invenio_rdm_records/records/mappings/os-v2/rdmrecords/records/record-v7.0.0.json

```diff
@@ -0,0 +1,1491 @@
+{
+  "settings": {
+    "index.query.default_field": [
```

Comment:
While reviewing this, me and @kpsherva were wondering if we are all ok with restricting the common search (without specifying the field(s)) to these fields. A quick discussion would be nice.

In any case, it looks like that the `text` fields have been added here, which makes sense. However these are missing:
```
dates.description <--- probably not needed
additional_descriptions.description
references.reference
additional_titles.title
```

Given that `pids` is not added, if we search for a DOI, will the record be found?